node/src/accountant/payment_adjuster/non_unit_tests/mod.rs

```diff
@@ -292,140 +251,240 @@ fn make_payable_account(
     }
 }
 
+fn generate_debt_age(gn: &mut ThreadRng, thresholds: &PaymentThresholds) -> u64 {
+    generate_range(
+        gn,
+        thresholds.maturity_threshold_sec,
+        thresholds.maturity_threshold_sec + thresholds.threshold_interval_sec,
+    ) / 2
+}
+
+fn generate_highly_randomized_payable_account_balance(
```

Comment:
During our discussion we realised that we can look for libraries or a built-in rust methodology to generate numbers with high entropy and we may not need to do calculations on our own to achieve that. If that's the case, you might want to replace this function with the newly found method.

This is something optional and should be done as per your choice, if you feel like it would be a good idea to work on it. Apart from that thanks for an interesting discussion.